ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/PartyRestrictionUserManagementAuthIT.scala

```diff
@@ -73,42 +72,51 @@ final class PartyRestrictionUserManagementAuthIT
         rights = Vector(
           uproto.Right(
             uproto.Right.Kind.IdentityProviderAdmin(uproto.Right.IdentityProviderAdmin())
-          ),
-          uproto.Right(uproto.Right.Kind.CanReadAs(uproto.Right.CanReadAs("some-party-2"))),
```

Comment:
This line should fail/pass depending on whether the party exists or not
Also the primary party below should exist

Reply:
the idea of this line is that it does not fail and just the IDPAdmin user is created. I had to remove this line as well as participant_admin who is acting here cannot create grant with missing party. 

"primary party below should exist" - I did not check for that, as `primary_party` is still being checked for permissions. So it is just a meta information, and in order to act with this party - you still need to have permission. Let me know if I am wrong here.